crates/re_space_view_spatial/src/scene/parts/cameras.rs

```diff
@@ -160,7 +131,11 @@ impl CamerasPart {
         let mut line_builder = scene_context.shared_render_builders.lines();
         let mut batch = line_builder
             .batch("camera frustum")
-            .world_from_obj(world_from_parent)
+            // The frustum is setup as a RUB frustum, but if the view coordinates are not RUB,
+            // we need to reorient the displayed frustum so that we indicate the correct orientation in the 3D world space.
```

Comment:
This is not correct. We should never "reorient" the displayed frustum. The camera extrinsics (transform3d) are the only thing that should be determining the orientation in world space.

The frustum is based on the mathematical definition of the projection matrix.  It must always point in either the positive or negative z dimension because that is the dimension along which the perspective projection is *defined* to happen.  The only thing we should be potentially changing based on view coordinates in the choice of whether we use +z or -z.